index.html

```diff
@@ -161,6 +163,42 @@ <h3 class="lang"><a href="#">Skills</a><i class="fas fa-angle-right"></i></h3>
 				<hr>
 			</section>
 		</section>
+
+
+
+
+
+
+
+
+
+		<section class="contact-form">
+			<h2>Contact me</h2>
+			<p>If you have an application you are interested in developing, a feature that you need built or a project
+				that
+				needs coding. I’d love to help with it </p>
+			<form action="https://formspree.io/f/myyljgwb" method="POST">
+				<fieldset>
+					<ul>
+						<li>
+								<input type="text" name="name" placeholder="Elyan" maxlength="30" required>
+						</li>
+						<li>
+								<input type="email" name="email" placeholder="Johndoe@mail.com" required>
+						</li>
+						<li>
+								<textarea name="message" placeholder="write your message here" maxlength="500"></textarea>
```

Comment:
- [ ] All the inputs must have the required attribute, you miss it here. Please add it

Reply:
I did have the required attribute to this, but web hint keeps flagging it as an error that it's not allowed(Linters provided as a requirement for the project) the reason I removed it, as I couldn't found a workaround without removing it.

Reply:
Got it at last.